codegen/src/utils/fetch_metadata.rs

```diff
@@ -201,14 +201,18 @@ async fn fetch_metadata(
         client: &impl ClientT,
         version: MetadataVersion,
     ) -> Result<Vec<u8>, FetchMetadataError> {
-        if !matches!(version, MetadataVersion::Version(14)) {
+        // If the user specifically asks for anything other than version 14 or "latest", error.
+        if !matches!(
+            version,
+            MetadataVersion::Latest | MetadataVersion::Version(14)
+        ) {
             return Err(FetchMetadataError::Other(
                 "The node can only return version 14 metadata using the legacy API but you've asked for something else"
```

Comment:
shouldn't this message say that it supports "v14" or "latest"?

Reply:
Only V14 can be provided from this legacy route, so whether they ask for V14 or "latest that node has" that's what they will get back, so I htink it's all good?

Reply:
ok, I see didn't understand that :P